docs/getting_started/installation.rst

```diff
@@ -60,6 +58,7 @@ line in `bundles.php` file of your project:
         Sonata\CoreBundle\SonataCoreBundle::class => ['all' => true],
         Sonata\BlockBundle\SonataBlockBundle::class => ['all' => true],
         Knp\Bundle\MenuBundle\KnpMenuBundle::class => ['all' => true],
+        Sonata\DoctrineORMAdminBundle\SonataDoctrineORMAdminBundle::class => ['all' => true],
```

Comment:
When reading the docs, it looks like the ORM bundle is required. Maybe add a sub chapter for the prefered implementation (orm, mongodb, ...)

Reply:
IMO we should totally remove that example and let user choose, I think the `tip` part is enough 

>Tip Don’t know which to choose? Most new users prefer SonataDoctrineORMAdmin, to interact with traditional relational databases (MySQL, PostgreSQL, etc).

Reply:
I agree on the part where this should be the choice for the user, but If you read the whole chapter, it makes sense. Because it is presenting all the options and then picking one to give an example on how to install SonataAdmin.

Reply:
Maybe just add a short comment above this line?